test/actions/channels.test.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import assert from 'assert';
+
+import * as Actions from 'actions/channels';
+import Client from 'client';
+import configureStore from 'store/configureStore';
+import {RequestStatus} from 'constants';
+import TestHelper from 'test_helper';
+
+describe('Actions.Channels', () => {
+    it('fetchMyChannelsAndMembers', (done) => {
+        TestHelper.initBasic(Client).then(() => {
+            const store = configureStore();
```

Comment:
We may want to consider freezing the stores when testing so that we can be sure we're not mutating anything. I'm not sure which part of the store you'd need to do that on though